packages/api/src/DAPI.js

```diff
@@ -51,7 +51,7 @@ class DAPI {
    * @param {string} indexName
    * @param {number} resultType
    * @param {Array<Buffer>} indexValuesList
-   * @param {Buffer} [startAtIdentifierInfo]
+   * @param {Object} [startAtIdentifierInfo]
```

Comment:
Why is that?

Reply:
Mistake in previous PR

Reply:
Documentation say it's must be object

Reply:
start_at_vote_poll_id_info | Object | No | Start poll ID information for pagination

Reply:
What is the object? What type is it?